container-cert-script/install-ca-cert.sh

```diff
@@ -34,6 +34,64 @@ copy_certs() {
     done
 }
 
+update_openssl() {
+    mkdir -p /tmp/ssl/ca-certificates/openssl
+    trust extract --format=openssl-directory --filter=ca-anchors --overwrite /tmp/ssl/ca-certificates/openssl
+    cp -R /tmp/ssl/ca-certificates/openssl/* /var/lib/ca-certificates/openssl/
+}
+
+update_etc_ssl() {
+    etccertsdir="/etc/ssl/certs"
+    pemdir="/var/lib/ca-certificates/pem"
+
+    trust extract --purpose=server-auth --filter=ca-anchors --format=pem-directory-hash -f /tmp/ssl/ca-certificates/pem
+    cp -R /tmp/ssl/ca-certificates/pem/* "$pemdir"/
+
+    # fix up /etc/ssl/certs if it's not a link pointing to /var/lib/ca-certificates/pem
+    if ! [ -L "$etccertsdir" -a "`readlink $etccertsdir`" = "../..$pemdir" ]; then
+        echo "Warning: $etccertsdir needs to be a link to ../..$pemdir, fixing" >&2
+        if [ -d "$etccertsdir" ]; then
+      mv -Tv --backup=numbered "$etccertsdir" "$etccertsdir.old"
+        fi
+        ln -Tsv --backup=numbered "../..$pemdir" "$etccertsdir"
+    fi
+}
+
+update_certbundle() {
+    cafile="/var/lib/ca-certificates/ca-bundle.pem"
+    cadir="/var/lib/ca-certificates/pem"
+
+    for i in "$@"; do
```

Comment:
I only see you calling this once and you're not passing any arguments?  Is there something about bash syntax I'm not understanding here?